WordPress/Classes/ViewRelated/Blog/QuickStartTours.swift

```diff
@@ -161,15 +161,15 @@ struct QuickStartShareTour: QuickStartTour {
     let title = NSLocalizedString("Social sharing", comment: "Title of a Quick Start Tour")
     let titleMarkedCompleted = NSLocalizedString("Completed: Social sharing", comment: "The Quick Start Tour title after the user finished the step.")
     let description = NSLocalizedString("Automatically share new posts to your social media accounts.", comment: "Description of a Quick Start Tour")
-    let icon = UIImage.gridicon(.share)
+    let icon = UIImage(named: "site-menu-social") ?? UIImage()
     let iconColor = UIColor.muriel(color: MurielColor(name: .blue, shade: .shade40)).color(for: UITraitCollection(userInterfaceStyle: .light))
     let suggestionNoText = Strings.notNow
     let suggestionYesText = Strings.yesShowMe
     let possibleEntryPoints: Set<QuickStartTourEntryPoint> = [.blogDetails]
 
     var waypoints: [WayPoint] = {
         let step1DescriptionBase = NSLocalizedString("Select %@ to continue", comment: "A step in a guided tour for quick start. %@ will be the name of the item to select.")
-        let step1DescriptionTarget = NSLocalizedString("Sharing", comment: "The menu item to select during a guided tour.")
+        let step1DescriptionTarget = NSLocalizedString("Social", comment: "The menu item to select during a guided tour.")
```

Comment:
I could've added namespacing, but since the rest of this file doesn't use it, I kept it consistent for now.